### DISCUSSION TITLE
Requesting bug fixes commits to master in a newer version release

### DISCUSSION THREAD

**Discussion by dev-pasa** [provectus/kafka-ui#4287] [General]
Hello Kafka-ui team, 
Thank you for your active community support. 

Requesting new version release:
The serde api bug was fixed in master branch after V0.7.1, which resolved the 404 error in the Topic page. 
Would it be possible to add the bug fixes and updates after June 27, in a newer release (v0.7.2)?

**Comment by vascoferraz** [comment]
@dev-pasa, what you are requesting is probably not going to happen because Kafka-UI development has stopped: https://github.com/provectus/kafka-ui/discussions/4255

So, if you are deploying this tool with Kubernetes I would suggest you to use the image tag **master**:

`helm upgrade --install kafka-ui kafka-ui/kafka-ui --version 0.7.5 -f "${TUTORIAL_HOME}/manifests/kafkaui-values.yaml" --set "image.repository=provectuslabs/kafka-ui,image.tag=master"`
